.github/workflows/ci-core.yml

```diff
@@ -357,7 +358,7 @@ jobs:
   clean:
     name: Clean Go Tidy & Generate
     if: ${{ !contains(join(github.event.pull_request.labels.*.name, ' '), 'skip-smoke-tests') && github.actor != 'dependabot[bot]' }}
-    runs-on: ubuntu22.04-8cores-32GB
+    runs-on: ubuntu-latest-8cores-32GB
```

Comment:
Only concern about switching to latest is that soon this will point to Ubuntu 24.04 (once it comes out of beta): https://github.com/actions/runner-images

Reply:
Ok I will move these down to `22.04`.